Resources/views/page_head_style.html.twig

```diff
@@ -5,6 +5,8 @@
     <link rel="stylesheet" type="text/css" href="{{ asset_url }}"/>
 {% endstylesheets %}
 
+<link rel="stylesheet" type="text/css" href="/bundles/ezplatformuiassets/vendors/flag-icon-css/css/flag-icon.min.css" />
```

Comment:
There is a `cssrewrite` filter for assetic, but it still requires second request to sever (paths must be corrected only for flags styles)... besides, it doesn't work as it should. I don't know how to solve it right now.